packages/lit-html/src/lit-html.ts

```diff
@@ -811,7 +811,7 @@ class TemplateInstance {
         nodeIndex++;
       }
     }
-    return fragment;
+    return (options?.creationScope ?? d).adoptNode(fragment);
```

Comment:
Here is where we used to call `customElements.upgrade()` too. I think we'll want some way to have a reference to a registry from the creationScope.

cc @kevinpschaaf @manolakis

Reply:
> used to call `customElements.upgrade()`

Oh, that might be why the tests are failing.